test/Application.UTest/Clans/Armory/BorrowClanArmoryCommandTest.cs

```diff
@@ -114,7 +114,7 @@ public async Task ShouldNotBorrowWithWrongClan()
     }
 
     [Test]
-    public async Task ShouldNotBorrowExistingItem()
+    public async Task ShouldBorrowExistingItem()
     {
```

Comment:
I do not think that you should be able to borrow already owned item. What is the use case for this?

Reply:
++